Automattic/wp-calypso
# Stats: Chart refresh- CSS styling

Related to https://github.com/Automattic/red-team/issues/250

## Proposed Changes

* Adds CSS styling to chart refresh + new chart header

## Why are these changes being made?


* As part of the chart refresh for the Date Filtering for the Traffic Page project

## Testing Instructions



* Open Calypso Live Branch
* Ensure that the chart on the traffic page still looks exactly the same as before. 
* Apply feature flag `stats/new-date-filtering`
* Ensure that the chart on the traffic page is now styled to look like the design screenshot below:

![image](https://github.com/user-attachments/assets/67dab2f2-a5e6-43c3-a6e8-836c820f94d3)


## Pre-merge Checklist
